
# SLIDE

<!-- badges: start -->
<!-- badges: end -->

The genome-wide association studies aim at identifying common or rare variants associated with common diseases and explaining more heritability. It is well known that common diseases are influenced by multiple single nucleotide polymorphisms (SNPs) that are usually correlated in location or function. In order to powerfully detect association signals, it is highly desirable to take account of correlations or linkage disequilibrium (LD) information among multiple SNPs in testing for association. Wang et al. (2019) proposed a test SLIDE that depicts the difference of the average multi-locus genotypes between cases and controls and derive its varianceâ€“covariance matrix in the retrospective design. This matrix is composed of the pairwise LD between SNPs. Thus SLIDE can borrow the strength from an external database in the population of interest with a few thousands to hundreds of thousands individuals to improve the power for detecting association. Extensive simulations show that SLIDE has apparent superiority over the existing methods, especially in the situation involving both common and rare variants, both protective and deleterious variants. For more detailed information, please refer to: Wang C, Deng S, Sun L, Li L, Hu Y-Q. A nonparametric test for association with multiple loci in the retrospective case-control study. Statistical Methods in Medical Research. 2020;29(2):589-602. doi:10.1177/0962280219842892. We optimized the source code of SLIDE to make it faster and better able to handle gene-based association analysis.